server/src/main/java/com/linecorp/centraldogma/server/internal/admin/authentication/LoginService.java

```diff
@@ -161,6 +176,18 @@ private UsernamePasswordToken usernamePassword(AggregatedHttpMessage req) {
         return throwResponseException("invalid_request", "request must contain username and password.");
     }
 
+    @Nullable
+    private AccessToken currentUserTokenIfPresent(UsernamePasswordToken usernamePassword) {
+        securityManager.authenticate(usernamePassword);
+        // Because securityManager.authenticate does not throw any Exception, the user is authenticated.
+        final AccessToken currentUserToken = cache.getIfPresent(usernamePassword.getUsername());
+        if (currentUserToken != null && currentUserToken.deadline() > System.currentTimeMillis()) {
```

Comment:
nit: Do we need to add the minimum remaining period for reusing the existing token? e.g.
```
currentUserToken.deadline() > System.currentTimeMillis() + 3600000
```

Reply:
An hour sounds OK to me. Just a minute should be OK?